### ISSUE TITLE
Autohome and endstops don't seem to be working.

### ISSUE THREAD

**Issue by DLMorrell** [makerbase-mks/MKS-Robin-Nano-V3.X#150]
Hello, I'm sure I have just messed this up somewhere and please tell me what all you need for this! I have an original Tronxy XY-3 that I'm working on changing over to this board but I've had some trouble. After plugging everything up and installing the firmware everything seems to work except for the endstops and autohome. When I connect via pronterface and send M119 it just says sending, but never gets a response from the printer, and when I click autohome there's a variety of issue. With autohome x it slightly moves the x axis, and that's it. With any other autohome the printer speeds all axis to the end spot really fast but the endstops don't trigger so the motors keep pushing and loudly grind. Like I said, I'm sure I just messed something up in the firmware, but I've tried everything I can find and could really use some help! Thank you in advance!

**Comment by DLMorrell** [comment]
I just found the section in the UI that lets you manually enter GCODE so I tried M119, and they are all showing triggered there. Looking into this, but help is still much appreciated.

**Comment by DLMorrell** [comment]
Ok I think I managed to figure this out, I don't have a bltouch on my printer and the initial firmware came with it setup for that. I went and disabled as many auto levelling things I could find, and then I just had to invert my endstops, and invert my x axis. Now it appears everything is working! I spent weeks on and off messing with this and just had to ask a question here to get my breakthrough, that's my life. I'll close this, and sorry for bothering!
